packages/domain-picker/src/utils.ts

```diff
@@ -40,3 +40,32 @@ export function isGoodDefaultDomainQuery( domainQuery: string ): boolean {
 		.replace( /[\u0300-\u036f]/g, '' )
 		.match( /[a-z0-9-.\s]/i );
 }
+
+/**
+ * Get the suggestions vendor
+ *
```

Comment:
Does this belong to the type?

Reply:
You're highlighting on a block of comment. Not sure what you mean? :)